drivers/media/i2c/lt6911uxc.c

```diff
@@ -658,14 +673,9 @@ static int lt6911uxc_get_fmt(struct v4l2_subdev *sd,
 	fmt->height = state->timings.bt.height;
 	fmt->field  = V4L2_FIELD_NONE;
 	fmt->colorspace = V4L2_COLORSPACE_SRGB;
+	fmt->ycbcr_enc = V4L2_YCBCR_ENC_601;
+	fmt->quantization = V4L2_QUANTIZATION_LIM_RANGE;
 
-	switch (fmt->code) {
```

Comment:
I'll agree that this switch/case is redundant as all conditions fall through the same code, but did it need to be changed?